tests/lava/lib/dnf/connect/test_connect.py

```diff
@@ -233,6 +242,44 @@ def _compute_weights(self) -> np.ndarray:
 
         self.assertTrue(np.array_equal(computed_weights, expected_weights))
 
+    def test_connection_is_sparse_for_none(self) -> None:
```

Comment:
I suggest renaming the test to be more readable. If test names can be read like sentences, you don't need an additional docstring.
What about `test_connection_is_sparse_when_no_connection_class_is_specified`?